### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ababazadeh**

### CODE REVIEW

**Comment by ababazadeh** [comment]
Adding other variants of demand for Chicago Regional test problem.

File: chicago-regional/README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ A large-scale, detailed representation of the Chicago region.
 ## Source
 Developed and provided by the Chicago Area Transportation Study (CATS)
 
+[Abbas Babazadeh](mailto:ababazadeh@ut.ac.ir) reports that two other credible versions of the Chicago Regional problem exist for which network data are identical to the first version given above, but demand matrices differ. The second version, known as FHWA 2009 project, has been provided by Michael Florian.The specific attributes of this version are as follows: total trips=1,429,901.18 ([click here](https://drive.google.com/file/d/1xVcL8G5C3xNeHMrsgIrgAPU5xWCnTaTn/view?usp=sharing) for demand data); optimul objective function value=17,016,655.2611360890; cost weights=0.0 (Toll and Distance). The Third version appears in "A Greedy Path-Based Algorithm for Traffic Assignment," Xie, J., Nie, Y., Liu, X., Transportation Research Record, Vol. 2672(48), pp. 33-44, 2018. For this version, total trips=1,429,895.88 ([click here](https://drive.google.com/file/d/13S3ThVdWCt8qCIYOknSHlGlqGJslWA6p/view?usp=sharing) for demand data), optimul objective function value=17,016,355.0629789010, and cost weights=0.0.
```
Comment (pedrocamargo) [inline chicago-regional/README.md:7]: I don't think it makes sense to have links to external sources here. It looks like something that will definitely break in the future.

Comment (pedrocamargo) [reply chicago-regional/README.md]: The issue of self citing is still there (minor issue), but the pull request should include the actual data. A link to somebody's google drive (for data that fits well in a git repo) is utterly inadequate

Comment (ababazadeh) [reply chicago-regional/README.md]: Thank you. The email address has been removed.

Comment (ababazadeh) [reply chicago-regional/README.md]: How can I resolve the next issue?

**Comment by ababazadeh** [comment]
Hi
Please let me know how can I resolve the problem?
thanks 
    On Tuesday, July 28, 2020, 03:48:18 AM GMT+4:30, Pedro Camargo <notifications@github.com> wrote:  
 
 


@pedrocamargo commented on this pull request.

In chicago-regional/README.md:
> @@ -4,6 +4,8 @@ A large-scale, detailed representation of the Chicago region.
 ## Source
 Developed and provided by the Chicago Area Transportation Study (CATS)
 
+[Abbas Babazadeh](mailto:ababazadeh@ut.ac.ir) reports that two other credible versions of the Chicago Regional problem exist for which network data are identical to the first version given above, but demand matrices differ. The second version, known as FHWA 2009 project, has been provided by Michael Florian.The specific attributes of this version are as follows: total trips=1,429,901.18 ([click here](https://drive.google.com/file/d/1xVcL8G5C3xNeHMrsgIrgAPU5xWCnTaTn/view?usp=sharing) for demand data); optimul objective function value=17,016,655.2611360890; cost weights=0.0 (Toll and Distance). The Third version appears in "A Greedy Path-Based Algorithm for Traffic Assignment," Xie, J., Nie, Y., Liu, X., Transportation Research Record, Vol. 2672(48), pp. 33-44, 2018. For this version, total trips=1,429,895.88 ([click here](https://drive.google.com/file/d/13S3ThVdWCt8qCIYOknSHlGlqGJslWA6p/view?usp=sharing) for demand data), optimul objective function value=17,016,355.0629789010, and cost weights=0.0.

The issue of self citing is still there (minor issue), but the pull request should include the actual data. A link to somebody's google drive (for data that fits well in a git repo) is utterly inadequate

File: chicago-regional/README.md (hunk 1 above)
Comment (pedrocamargo) [reply chicago-regional/README.md]: The changes in the text need to go here: https://github.com/bstabler/TransportationNetworks/tree/master/chicago-regional, and not in the main readme file.

And add the two demand files to the Pull request (in the appropriate folder) as well.

**Comment by ababazadeh** [comment]
"And add the two demand files to the Pull request (in the appropriate folder." I could not follow this task. Which folder?
    On Tuesday, July 28, 2020, 05:14:33 AM GMT+4:30, Pedro Camargo <notifications@github.com> wrote:  
 
 


@pedrocamargo commented on this pull request.

In chicago-regional/README.md:
> @@ -4,6 +4,8 @@ A large-scale, detailed representation of the Chicago region.
 ## Source
 Developed and provided by the Chicago Area Transportation Study (CATS)
 
+[Abbas Babazadeh](mailto:ababazadeh@ut.ac.ir) reports that two other credible versions of the Chicago Regional problem exist for which network data are identical to the first version given above, but demand matrices differ. The second version, known as FHWA 2009 project, has been provided by Michael Florian.The specific attributes of this version are as follows: total trips=1,429,901.18 ([click here](https://drive.google.com/file/d/1xVcL8G5C3xNeHMrsgIrgAPU5xWCnTaTn/view?usp=sharing) for demand data); optimul objective function value=17,016,655.2611360890; cost weights=0.0 (Toll and Distance). The Third version appears in "A Greedy Path-Based Algorithm for Traffic Assignment," Xie, J., Nie, Y., Liu, X., Transportation Research Record, Vol. 2672(48), pp. 33-44, 2018. For this version, total trips=1,429,895.88 ([click here](https://drive.google.com/file/d/13S3ThVdWCt8qCIYOknSHlGlqGJslWA6p/view?usp=sharing) for demand data), optimul objective function value=17,016,355.0629789010, and cost weights=0.0.

The changes in the text need to go here: https://github.com/bstabler/TransportationNetworks/tree/master/chicago-regional, and not in the main readme file.

And add the two demand files to the Pull request (in the appropriate folder) as well.

**Comment by pedrocamargo** [comment]
Abbas, how come?  Haven't you seen that there is a folder for each model when you clone the repository to your computer?

**Comment by ababazadeh** [comment]
Hi. really nothing. 
I have got a repo named "ababazadeh/TransportationNetworks" folked form "bstabler/TransportationNetworks" .Should I add the demand files to this repo? if yes, the git does not allow saying that they are larger than 25 MB.   
could you please direct me with more details?
thanks 
    On Tuesday, July 28, 2020, 08:34:27 AM GMT+4:30, Pedro Camargo <notifications@github.com> wrote:  
 
 


Abbas, how come? Haven't you seen that there is a folder for each model when you clone the repository to your computer?

**Pull request closed without merging**